Formula/lighthouse.rb

```diff
@@ -0,0 +1,35 @@
+class Lighthouse < Formula
+  desc "Rust Ethereum 2.0 Client"
+  homepage "https://github.com/sigp/lighthouse"
+
+  url "https://github.com/sigp/lighthouse.git",
+      tag:      "v1.4.0",
+      revision: "3b600acdc5bf9726367c18277a22486573b8b457"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "rustup-init" => :build
```

Comment:
This needs to build with the `rust` formula.

Reply:
I may be reading something wrong but I don't think it’s possible to install a specific version of rust compiler with homebrew `rust` formula. Is that correct?

The available `rust` version `1.53.0` came out after lighthouse `1.4.0` and made it impossible to compile:
https://github.com/sigp/lighthouse/issues/2420

Reply:
That sounds like a blocker for this PR (formula cannot build with the latest Rust)

Reply:
[`1.5.0`](https://github.com/sigp/lighthouse/releases/tag/v1.5.0) has been released, so this may be unblocked